src/components/form/form.component.js

```diff
@@ -31,6 +31,8 @@ const Form = ({
   const formRef = useRef();
   const formFooterRef = useRef();
 
+  const renderFooter = !!(saveButton || leftSideButtons || rightSideButtons);
```

Comment:
Yeah good point, I'll add a check in for them as well. 
It probably shouldn't be possible to set those props without passing a save button in, but if we start enforcing that now it would be a breaking change, so not worth it for the purposes of this PR.